src/app/+detail/index.ts

```diff
@@ -20,8 +20,9 @@ export const routes = [
     BrowserModule,
     FormsModule,
     RouterModule.forChild(routes),
+    MdModule.forRoot()
```

Comment:
@zigzag95 i wasnt sure about that for a lazily loaded module, thanks for the tip! updating the pr now...